extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudStreamExtractor.java

```diff
@@ -196,6 +197,10 @@ public String getHlsUrl() throws ParsingException {
             throw new ExtractionException("Could not get SoundCloud's track audio url", e);
         }
 
+        if (audioStreams.isEmpty()) {
+            throw new ContentNotSupportedException("HLS audio streams / opus streams are not yet supported");
```

Comment:
Opus is with a capital O, however I'd suggest to not mention that at all, but just HLS, since the tracks that have Opus HLS streams, also have MP3 HLS streams.